usr/bin/whonix-gateway-firewall

```diff
@@ -116,6 +116,26 @@ variables_defaults() {
     fi
   fi
 
+  if [ -z "${NON_TOR_GATEWAY_IP6:-}" ]; then
+    if test -f /usr/share/qubes/marker-vm; then
+      NON_TOR_GATEWAY_IP6=""
+    else
+      ## 10.0.2.2/24: VirtualBox DHCP
+      ## IP HARDCODED. If you want to change IP, set variable GATEWAY_IP through a
+      ## drop-in configuration snippet in /etc/whonix_firewall.d
+      ## configuration folder instead.
+      NON_TOR_GATEWAY_IP6="\
+            ::1/128 \
+            fd19:c33d:88bc::0/96 \
+            ::ffff:127.0.0.0/120 \
```

Comment:
I see why you set this block to have a `/120` prefix, but I'm fairly certain `::ffff:127.0.0.0/120` should be `::ffff:127.0.0.0/104` since the entire `127.0.0.0/8` block is reserved for loopback addresses. That probably also means that the corresponding IPv4 block (`127.0.0.0/24`) should be changed to `127.0.0.0/8` too. @adrelanos you're better at networking than I am, thoughts?

Reply:
Agreed, I've updated it accordingly. It's unusual to see an address outside of 127.0.0.0/24, but it should be the entire /8 network.